spec/models/account/field_spec.rb

```diff
@@ -68,6 +68,14 @@
         end
       end
 
+      context 'with an HTTP URL' do
```

Comment:
The added coverage here (http vs https urls) seems useful on its own, but not related to the actual change here (non-ascii chars in url values)...?